app/views/layouts/application.html.erb

```diff
@@ -16,7 +16,7 @@
     <main class="px4">
       <div class="flash-message">
         <% flash.each do |name, msg| %>
-          <%= content_tag :div, msg, class: name %>
+        <%= content_tag :div, sanitize(msg), class: name %>
```

Comment:
remind me what the sanitize does again? I didn't deal with flash messages too much in past projects...